src/parameterizations/gravity_wave_drag/orographic_gravity_wave_helper.jl

```diff
@@ -1,5 +1,57 @@
 using NCDatasets
 using Interpolations
+using Statistics: mean
+
+"""
+    calc_hmax_latlon(elev, lon, lat)
+
+    Calculated hmax used in orographic gravity wave
+    - elev: surface elevation
+    - lon: longitude
+    - lat: latitude
+"""
+function calc_hmax_latlon(elev, lon, lat, earth_radius)
+    println("compute hmax")
+    FT = eltype(elev)
+
+    elev[elev .< FT(0)] .= FT(0)
+
+    dlat = lat[2] - lat[1]
+    dlon = lon[2] - lon[1]
+
+    scale =
+        sind(40) ./ earth_radius ./ sind.(max.(FT(20), abs.(lat))) .* FT(100e3) # FT(200e3) is used in the codes sent by Steve Garner
+
+    # compute weights for the spatial running mean using the Blackman kernel
+    ilat_range = Int.(round.(scale ./ deg2rad(dlat)))
+    ilon_range =
+        min.(
+            Int.(round.(scale ./ (deg2rad(dlon) .* cosd.(lat)))),
+            Int(round(length(lon) / 16)),
+        )
+
+    hmax = zeros(size(elev))
+    for (i, ilon) in enumerate(lon)
+        if i % 10 == 1
+            println("i = $i, and ilon = $ilon")
```

Comment:
Is this print intentional?

Reply:
oops.. for debugging purposes only. good catch!